### PR TITLE
Feature/ROC-5382 update ccj confirmation page

### PR DESCRIPTION
**Pull request by BenDoherty0103**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5382

### Change description
Content changes concerning when a claimant requests a CCJ

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/ccj/views/confirmation-ccj.njk [hunk 1]
```diff
@@ -17,10 +17,11 @@
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
 
-      <p>{{ t('You requested a County Court Judgment on {{ ccjRequestedAt }}. We’ll contact you within 14 days to tell you whether the judgment has been entered.',
+      <p>{{ t("You requested a County Court Judgment on {{ ccjRequestedAt }}. We'll contact you within 5 days to tell you whether the judgment has been entered.",
```
Comment (sabahirfan) [inline src/main/features/ccj/views/confirmation-ccj.njk]: Please use the correct apostrophe and single quotes here. Please revert to the existing code for this.

File: src/main/features/ccj/views/confirmation-ccj.njk [hunk 2]
```diff
@@ -17,10 +17,11 @@
...
           { ccjRequestedAt: ccjRequestedAt | date }) }}
       </p>
-      <p>{{ t('{{ defendantName }} can no longer respond to the claim using this service - they may have responded by post.', { defendantName: defendantName }) }}</p>
+      <p>{{ t('{{ defendantName }} can no longer respond to the claim using this service.', { defendantName: defendantName }) }}</p>
+      <p>{{ t("If we receive a postal response that they sent before the deadline we'll reject your request for judgment.") }}</p>
```
Comment (sabahirfan) [inline src/main/features/ccj/views/confirmation-ccj.njk]: Same here

File: src/main/features/dashboard/views/macro/claimStatus/ccjEligible.njk [hunk 3]
```diff
@@ -11,12 +11,12 @@
     <p>{{ t('{{ defendantName }} has not responded to your claim by the deadline. '
         + 'You can request a County Court Judgment (CCJ) against them.',
         { defendantName : defendantName }) }}</p>
-    <p>{{ t('The court will make an order asking them to pay the money. It does not guarantee that they pay it.') }}</p>
+    <p>{{ t("The court will order them to pay the money. It doesn't guarantee that they'll pay it.") }}</p>
```
Comment (sabahirfan) [inline src/main/features/dashboard/views/macro/claimStatus/ccjEligible.njk]: Same here

Comment (BenDoherty0103) [reply src/main/features/dashboard/views/macro/claimStatus/ccjEligible.njk]: Thanks Sabah! Will run through these now.

File: src/main/features/ccj/views/confirmation-ccj.njk (hunk 1 above)
Comment (andrewwa-kainos) [reply src/main/features/ccj/views/confirmation-ccj.njk]: option + shift + ]

**Pull request merged**